test/e2e/staking.js

```diff
@@ -50,106 +49,33 @@ test("staking", async function(t) {
+    // 0px;">...</div>
+    await sleep(500)
 
-    // wait until the validators are showing again
-    await app.client.waitForVisible("#go-to-bonding-btn", 30000)
-    // TODO: re-enable once we've added back optimistic updates
-    // t.equal(
-    //   await app.client.$(".li-validator__value.your-votes").getText(),
-    //   bondedStake.toString(),
-    //   "Delegate steak in validator updated correctly"
-    // )
-
-    t.end()
-  })
-
-  t.test("unbonding", async function(t) {
-    // validator should already be in the cart so we only need to click a button to go to the bonding view
-    await app.client.$("#go-to-bonding-btn").click()
-    // the unbonded atoms are equal to the total owned stake minus the bonded atoms (150 - 100 == 50)
-    //TOD: re-enable after optimistic loading (or actual loading) update
-    // t.ok(
-    //   await waitForValue(
-    //     () => app.client.$("#new-unbonded-atoms"),
-    //     (totalUserStake - bondedStake).toString()
-    //   ),
-    //   "Left over steak shows correctly"
-    // )
-    // the bonded amount should show 120
-    // t.equal(
-    //   await app.client.$(".bond-candidate .bond-value__input").getValue(),
-    //   bondedStake.toString(),
-    //   "Candidate bond matches current bond"
-    // )
-
-    // the bonded candidate is lowered by 20 stake
-    // this is reflected correctly in the unbonded atoms section (50 - 20 === 30)
     await app.client
-      .$(".bond-candidate .bond-value__input")
-      .setValue(bondedStake - 20)
-    //TOD: re-enable after optimistic loading (or actual loading) update
-    // t.equal(
-    //   await app.client.$("#new-unbonding-atoms").getValue(),
-    //   (20).toString(),
-    //   "Unbonding steak shows correctly"
-    // )
-
-    await app.client.$("#btn-bond").click()
-
-    // should fail
-    t.ok(await app.client.isVisible(".tm-form-msg--error"), "shows error")
-
-    await app.client.$("#bond-confirm").click()
-    await app.client.$("#btn-bond").click()
-
-    t.ok(!(await app.client.isVisible(".tm-form-msg--error")), "hides error")
-
-    bondedStake -= 20
-
-    // wait until the validators are showing again
-    // await app.client.waitForVisible("#go-to-bonding-btn", 30000)
-    // TODO: Update when we fix the optimistic ui changes again
-    // t.equal(
-    //   await app.client.$(".li-validator__value.your-votes").getText(),
-    //   bondedStake.toString(),
-    //   "Delegate steak in validator updated correctly"
-    // )
-
+      .click(`//button/*[. = 'Stake']`)
+      .setValue(`#amount`, 10)
+      .click(`//*[@id = 'modal-stake']//button//*[. = 'Stake']`)
+      .waitForVisible(
+        `//*[. = 'You have successfully staked your Steaks.']`,
+        5 * 1000
+      )
+
+      // Go back to Staking page.
+      .click(`//a//*[. = 'Staking']`)
+
+    // Why is this necessary?  See
+    // https://github.com/jprichardson/tape-promise#example-asyncawait
```

Comment:
did you try it without?

Reply:
Yes, it hung.

Reply:
How will the comment help then? ;P

Reply:
It's not technical debt because it doesn't need to be fixed but it's a note to the next programmer that something unexpected is happening.